app/dashboard/models.py

```diff
@@ -2390,13 +2416,21 @@ def absolute_url(self):
 
     @property
     def username(self):
-        handle = ''
```

Comment:
Nice change! Is there a case where both if statements return false and we end up returning `None` here?

Reply:
Oh not that I've seen of but it's safe to add it! nice catch 🙌

Reply:
handled @danlipert  ^_^